cEP-0010.md

```diff
@@ -139,7 +142,17 @@ Here is the list of aspects to be implemented along with their corresponding
 - `Root.Formatting.Quotation`: This aspect checks on the quotations being used
   in your files given `Root.Formatting.Quotation.preferred_quotation`.
 - `Root.Formatting.Quotation.preferred_quotation`: This taste represents the
-  user's preferred quotation.
+  preferred quotation marks for the codebase. It ensures that every string
+  contains the preferred quotes.
+  For example, `"This is double quoted"` is changed to `'This is double quoted'`
+  if `preferred_quotation` is `'`.
+- `Root.Formatting.Quotation.force_preferred_quotation`: This taste represents
+  whether preferred quotes are absolutely compulsory to adhere (even in cases
+  where other quotation marks are present inside string containing the
+  preferred quotes).
```

Comment:
This taste represents whether the preferred quotes are compulsory to adhere to even in cases where other quotation marks are present inside a string containing the preferred style of quotes